jsons/Buildings.json

```diff
@@ -74,12 +79,29 @@
 		"uniques": [
 			"Only available <if [Vegas Casino] is constructed in all [non-[Puppeted]] cities>",
 			"Cost increases by [30] per owned city",
-			"Can only be built <in [Capital] cities>"
+			"Can only be built <in [Capital] cities>",
 			"Only available <in cities without a [The Tops]>",
-			"Only available <in cities without a [Gomorrah]>",
+			"Only available <in cities without a [Gomorrah]>"
 		],
 		"requiredTech": "Currency"
 	},
```

Comment:
I would add these for the replacements of the congress and council. (I might change them depending on if there are better options for the congress and council replacements.) Other than that everything else is good.
	{
		"name": "Triple Advisatory Council",
		"uniqueTo": "New Vegas",
		"replaces": "Congress",
		"cost": 100,
		"requiredTech": "Civil Service",
		"uniques": [
			"[+1 Culture, +1 Happiness, +1 Food] per [9] population [in all cities]",
			"[+5]% growth [in all cities]",
			"[-5]% Unhappiness from [Population] [in all cities]",
			"[+5]% Production when constructing [All] buildings [in this city]",
			"Only available <if [Search for Survivors] is constructed>",
			"Can only be built <in [Capital] cities>",
			"Only available <in cities without a [Chairman Advisatory Council]>",
			"Only available <in cities without a [Omertas Advisatory Council]>",
			"Only available <in cities without a [White Glove Advisatory Council]>",
			"Destroyed when the city is captured"
		]
	},
	{
		"name": "Chairman Advisatory Council",
		"uniqueTo": "New Vegas",
		"replaces": "Council",
		"cost": 50,
		"requiredTech": "Civil Service",
		"uniques": [
			"[+3 Culture] per [10] population [in all cities]",
			"[+15]% Production when constructing [All] buildings [in this city]",
			"Only available <if [Search for Survivors] is constructed>",
			"Can only be built <in [Capital] cities>",
			"Only available <in cities without a [Omertas Advisatory Council]>",
			"Only available <in cities without a [White Glove Advisatory Council]>",
			"Only available <in cities without a [Triple Advisatory Council]>",
			"Destroyed when the city is captured"
		]
	},
	{
		"name": "Omertas Advisatory Council",
		"uniqueTo": "New Vegas",
		"cost": 50,
		"requiredTech": "Civil Service",
		"uniques": [
			"[+3 Happiness] per [10] population [in all cities]",
			"[+15]% growth [in all cities]",
			"Only available <if [Search for Survivors] is constructed>",
			"Can only be built <in [Capital] cities>",
			"Only available <in cities without a [Chairman Advisatory Council]>",
			"Only available <in cities without a [White Glove Advisatory Council]>",
			"Only available <in cities without a [Triple Advisatory Council]>",
			"Destroyed when the city is captured"
		]
	},
	{
		"name": "White Glove Advisatory Council",
		"uniqueTo": "New Vegas",
		"cost": 50,
		"requiredTech": "Civil Service",
		"uniques": [
			"[+3 Food] per [10] population [in all cities]",
			"[-15]% Unhappiness from [Population] [in all cities]",
			"Only available <if [Search for Survivors] is constructed>",
			"Can only be built <in [Capital] cities>",
			"Only available <in cities without a [Chairman Advisatory Council]>",
			"Only available <in cities without a [Omertas Advisatory Council]>",
			"Only available <in cities without a [Triple Advisatory Council]>",
			"Destroyed when the city is captured"
		]
	},